### PR TITLE
Adding linting to Webpack and configuring absolute import pathing

### PR DESCRIPTION
**Pull request by itstang**
- Adding lint checks for webpack
- Adding script to run server via `yarn start` with auto compiling and reloading
- Adding ability to use absolute pathing instead of relative pathing:

	-	```js
		// old
		import App from "./components/app"
		//new
		import App from "components/app"
		```
		```js
		// old
		import Layout from "../../components/ui/layout"
		// new
		import Layout from "components/ui/layout"
		```

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+NODE_PATH=./
```
Comment (KeijiBranshi) [inline .env]: what this do?

File: webpack.config.js [hunk 2]
```diff
@@ -23,7 +24,7 @@ module.exports = {
     new HtmlWebpackPlugin({
       inject: false,
       template: require("html-webpack-template"),
-      appMountId: "root"
-    })
-  ]
-}
\ No newline at end of file
+      appMountId: "root",
```
Comment (KeijiBranshi) [inline webpack.config.js]: is there a rule we have in place to enforce these trailing commas?

File: tsconfig.json [hunk 3]
```diff
@@ -14,6 +14,11 @@
     "removeComments": true,
     "sourceMap": true,
     "strictNullChecks": true,
-    "target": "es6"
-  }
+    "target": "es6",
+    "baseUrl": ".",
+    "paths": {
+      "components/*": ["./src/components/*"]
+    }
```
Comment (KeijiBranshi) [inline tsconfig.json]: not sure if this is allowed, but should we have trailing commas in the tsconfig too?

File: .env (hunk 1 above)
Comment (itstang) [reply .env]: this sets up your local node environment to understand where the modules are located based on an absolute path when importing node modules, not technically required for our use case but could be a nice to have

File: webpack.config.js (hunk 2 above)
Comment (itstang) [reply webpack.config.js]: there is as part of the `prettier` eslint rules as well as it being the only thing configured in prettier 😂 

https://github.com/MichaelQ94/rcubed/blob/a8c605cd0d59f342f54270a4b6c6929a9fcc0647/.prettierrc#L2

File: tsconfig.json (hunk 3 above)
Comment (itstang) [reply tsconfig.json]: not technically allowed since that's malformed json, trailing commas should only be in js/ts/tsx files

**Pull request merged**